_introduction.md

```diff
@@ -1,22 +1,16 @@
 # Introduction
 
-Welcome to the Uphold API, we're glad to have you! Provided here is all the documentation you need to help you create custom and revolutionary new services powered by the Uphold Platform. With your ingenuity, together we can serve the needs of individuals and organizations across the globe and change the financial services ecosystem forever.
+Welcome to the Uphold API — we're glad to have you! Here you can find all the documentation you need to help you create custom and revolutionary new services powered by the Uphold Platform. With your ingenuity, together we can serve the needs of individuals and organizations across the globe and change the financial services ecosystem forever.
```

Comment:
Here I would either go with: `Together, with your ingenuity, we...` or `With your ingenuity; together....

Reply:
I agree. To be honest I'd rewrite this entire sentence to be a little less grandiose, but one step at a time ;)

Reply:
Indeed. :)